apps/cookbook/src/app/examples/radio-example/examples/states.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-radio-states-example',
+  template: `<kirby-radio-group>
+  <kirby-radio text="Default"></kirby-radio>
+</kirby-radio-group>
+
+<kirby-radio-group value="bacon">
+  <kirby-radio value="bacon" text="Checked"></kirby-radio>
+</kirby-radio-group>
+
+<kirby-radio-group>
+  <kirby-radio disabled="true" text="Disabled"></kirby-radio>
+</kirby-radio-group>
+
+<kirby-radio-group value="bacon">
+  <kirby-radio disabled="true" value="bacon" text="Disabled checked"></kirby-radio>
+</kirby-radio-group>
+
+<kirby-radio-group>
+  <kirby-radio hasError="true" text="Has error"></kirby-radio>
```

Comment:
Consider renaming prop to message to be consistent with form-field
```suggestion
  <kirby-radio hasError="true" message="Has error"></kirby-radio>
```

Reply:
If it's not used for error messages, I would properly prefer label as it's a form-control?

Reply:
It's not a message, it's the text/label if the radio button.
I was considering using `label` but that would be incosistent with every other Kirby component, where we use the term `text`